packages/doltgres/content/reference/sql/version-control/dolt-system-tables.md

```diff
@@ -227,14 +222,20 @@ as a result, and manually inserting a stored procedure must also have
 a lowercase `name`. Otherwise, it will be invisible to some
 operations, such as `DROP PROCEDURE`.
 
-### Example Query
+#### Example Query
 
 ```sql
-CREATE PROCEDURE simple_proc1(x DOUBLE, y DOUBLE) SELECT x*y;
-CREATE PROCEDURE simple_proc2() SELECT name FROM category;
-```
+postgres=> CREATE PROCEDURE simple_proc1(x DOUBLE, y DOUBLE) AS SELECT x*y;
```

Comment:
This doesn't actually work, since stored procedures don't work yet. Probably just remove for now